src/vos/tests/vts_aggregate.c

```diff
@@ -611,6 +618,8 @@ discard_1(void **state)
 			    ds.td_agg_epr.epr_lo, ds.td_iod_size);
 		aggregate_basic(arg, &ds, 0, NULL);
 	}
+
+	cleanup();
```

Comment:
I don't see the purpose of this call for each discard/aggregation test case.

Reply:
I wanted to assert that the aggregation entries get added to the pool heap.   Running the garbage collector after each aggregation test allows me to test this.

Reply:
I see, but most aggregation/discard test cases won't add anything to GC. Maybe only object/key punch involved tests adds things to GC.